dashboard/app/models/pd/workshop.rb

```diff
@@ -88,6 +88,12 @@ class Pd::Workshop < ActiveRecord::Base
       SUBJECT_CSP_WORKSHOP_2 = 'Units 2 and 3: Processing data, Algorithms, and Programming',
       SUBJECT_CSP_WORKSHOP_3 = 'Units 4 and 5: Big Data, Privacy, and Building Apps',
       SUBJECT_CSP_WORKSHOP_4 = 'Units 5 and 6: Building Apps and AP Assessment Prep'
```

Comment:
We could leverage [ActiveRecord enums](http://api.rubyonrails.org/classes/ActiveRecord/Enum.html) to normalize these strings in the DB. That would also open the door to i18n, since there would be a string lookup anyway.